### PR TITLE
Updated Readme File

### PR DESCRIPTION
**Pull request by i-m-karanagarwal**
Added the commands in readme.md to install the dependencies.
This closes issue #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,32 +21,41 @@ If you wish to contribute, we highly recommend following the guidelines mentione
    git clone https://github.com/your_username/devmate.git
 ```
 
-**3.** Navigate to the project directory.
+**3.** Install Dependencies.
+```
+   npm install next react react-dom
```
Comment (AnkitaMalik22) [inline README.md:26]:
```suggestion
   npm i
```

Comment (AnkitaMalik22) [reply README.md:26]: we can add npm i to install all the dependencies!

Comment (AnkitaMalik22) [reply README.md:26]: same for yarn and pnpm !

**Review by AnkitaMalik22 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (i-m-karanagarwal) [reply README.md]: Done

**Review by AnkitaMalik22 — APPROVED** [review]

**Comment by AnkitaMalik22** [comment]
Thanks for contributing!

**Pull request merged**